projects/zstd/project.yaml

```diff
@@ -4,7 +4,12 @@ auto_ccs:
   - "Yann.collet.73@gmail.com"
   - "terrelln@fb.com"
   - "cyan@fb.com"
+fuzzing_engines:
+  - libfuzzer
+  - afl
+  - dataflow
```

Comment:
I don't like that someone must specify "dataflow" for fuzzing_engine and sanitizers, maybe we can simply ignore the sanitizer argument when engine dataflow is used instead? Or how about using a "none" sanitizer

Reply:
Yeah, I'm also not quite happy with this, but otherwise I'd have to hack something in the travis script, like make `sanitizer` value optional, then invoke `helper.py` without `--sanitizer` and let it use the default value. Don't like going that path either, as dataflow is actually both a new engine (we link against `DataFlow.o`) and a new sanitizer (we use different compilation flags), so it feels like specifying `dataflow` twice in `project.yaml` is the lesser of the evils.

Reply:
//cc @oliverchang, in case he's got a better idea on how to indicate a new engine and new sanitizer at once, just discussed this with him

Reply:
Why not just make a "none" sanitizer and use that in travis?

Reply:
I think `none` sanitizer would be incorrect, as one would expect it not to have any sanitizer flags while in fact it enables several compiler flags: https://github.com/google/oss-fuzz/blob/c60fcd7247da0505e3977b0725d795a96966b3db/infra/base-images/base-builder/Dockerfile#L38

So it's clearly a new sanitizer in our model, it's just not compatible with other fuzzing engines right now, like MSan is not compatible/used with AFL, etc.

Reply:
Shouldn't these be coverage flags rather than sanitizer flags? This seems easy to answer for "-fsanitize-coverage" flags, I think its fine too for "-fsanitize=dataflow" since it seems to have more in common with coverage flags than other kinds of sanitizer flags IMO

Reply:
I think having both engine and sanitizer is fine here, since that's actually what's happening here. Hacking around it most likely won't be very nice.

+1 to putting the coverage flags in the right place, but I'm not sure about -fsanitize=dataflow.

Reply:
That's a good point, I don't recall why I've added those here, must be a mistake. Moved coverage instrumentation flags into `COVERAGE_FLAGS_dataflow`, `-fsanitize=dataflow` is in `SANITIZER_FLAGS_dataflow`. Thanks!